shared/js/settings.js

```diff
@@ -1,143 +1,143 @@
-var load = require('load');
+var load = require('./load');
```

Comment:
Any reason you're using `var load` here and in site.js, but `const load` elsewhere? Same for `var utils` in site.js and `var settings` in trackerLists.js among others?

Reply:
That's a good point. I was wondering whether to make them all `consts`s here or elsewhere. Lemme just change them to `const`.